tests/test_actions.py

```diff
@@ -397,6 +399,19 @@ def test_launch_with_stack_that_failed_to_create(
         mock_create.assert_called_once_with()
         assert response == sentinel.launch_response
 
+    @patch("sceptre.plan.actions.StackActions.create")
+    @patch("sceptre.plan.actions.StackActions.delete")
+    @patch("sceptre.plan.actions.StackActions._get_status")
+    def test_launch_with_stack_that_does_not_exist(
+        self, mock_get_status, mock_delete, mock_create
+    ):
+        mock_get_status.return_value = "REVIEW_IN_PROGRESS"
```

Comment:
Are lines 408 and 383 swapped? 

```
380: def test_launch_with_stack_in_review_in_progress( ...
...
383: mock_get_status.side_effect = StackDoesNotExistError()
```

```
405:  def test_launch_with_stack_that_does_not_exist(
...
408: mock_get_status.return_value = "REVIEW_IN_PROGRESS"
```

Reply:
This is actually coming from one of the PRs merged into this integration branch, and it's just a quirk of how GitHub is rendering the diffs:

<img width="1202" alt="Screenshot 2024-06-12 at 11 45 03 am" src="https://github.com/Sceptre/sceptre/assets/2305330/dff29a4a-8e9c-4d62-8b3f-271ed57aca2e">